vip-mail.php

```diff
@@ -12,13 +12,14 @@ class VIP_SMTP {
 	function init() {
 		add_action( 'phpmailer_init',    array( $this, 'phpmailer_init' ) );
 		add_action( 'bp_phpmailer_init', array( $this, 'phpmailer_init' ) );
+
+		add_filter( 'wp_mail_from', array( $this, 'filter_wp_mail_from' ) );
```

Comment:
What about setting this to priority 1 so it hits earlier, so other code can still override without worrying about priorities?